### ISSUE TITLE
Unable to open notebooks from desktop on Windows

### ISSUE THREAD

**Issue by jenren** [nteract/archived-desktop-app#4475]
Hi! I'm new to filing issues on GitHub, so please let me know if there's any additional information I can provide to be helpful. Thanks in advance to y'all for building an awesome product :) 

**Application or Package Used**
nteract desktop (0.14.3)

**Describe the bug**
When trying to open a notebook by clicking on it from my local machine, I get an error that blocks me from loading my notebook in nteract. I uninstalled and reinstalled the latest version of nteract desktop for Windows (0.14.3) but still encounter the same error messages: "No kernels installed" and "A JavaScript error occurred in the process." For context, I have Anaconda installed, and I have the IPython kernel installed. I wonder if I'm hitting this issue: #3670?

**To Reproduce**

1. Click on local notebook 
2. The 2 error messages appear (copied below)
3. Notebook does not open in nteract

**Expected behavior**
I expect the notebook to load normally in nteract, but at this point I can't get to the main view where I can specify a kernel.

**Screenshots**
![image](https://user-images.githubusercontent.com/5131960/60374130-bb2f1280-99b7-11e9-89f9-a1cc86e280f2.png)
![image](https://user-images.githubusercontent.com/5131960/60374140-c3874d80-99b7-11e9-9948-32c4725fa1b9.png)

**Desktop**

- OS: Windows 10
- Browser: Chrome

**Comment by captainsafia** [comment]
Thanks for opening this issue, @jenren! Let's see if we can help you resolve it.

Do you have Jupyter installed on your machine?

**Comment by jenren** [comment]
Hi @captainsafia! Yep, checking my base conda environment and verified that I've got Jupyter installed.

**Comment by hydrosquall** [comment]
Thanks for the detailed report, @jenren ! Two small debugging questions:

1. Prior to installing nteract, did you add the ipython kernel for python3 using the same steps as the [docs page](https://nteract.io/kernels), or did you have ipython installed for some previous work? Additionally, was this done as a global install or using a virtual environment?
2. Did the temporary workaround from the linked thread of invoking `start nteract.exe` (https://github.com/nteract/nteract/issues/3670#issuecomment-482777314) help? While it's not ideal, hopefully that could unblock you while we figure out where the javascript error is coming from.

I have a Windows laptop that I haven't used in a while that could be used to try to reproduce this, so I'd want to make sure I was recreating the same conditions.

**Comment by willingc** [comment]
I'm going to close this issue as stale since there have been a number of releases and improvements since the initial report. If there are still user questions, please feel free to file a new issue :sunny:
